client/src/client_v2/HanabiCard.ts

```diff
@@ -1,3 +1,8 @@
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+// TODO: fix typing issues in client_v2
```

Comment:
I skipped client_v2 on purpose; it doesn't seem to be working anyway, and I think it should be gutted out

Reply:
thats fine